app/services/SubmissionService.scala

```diff
@@ -49,4 +49,19 @@ class SubmissionService @Inject()(
       case _ =>
         Future.successful(INTERNAL_SERVER_ERROR)
     }
+
+  def submitDucrDisassociation(
```

Comment:
I'll just leave a question here, if we shouldn't have a separate, new service for all consolidations? It'd separate the logic.

Reply:
Given theres only going to be 4 methods in here I'm not sure theres any value in splitting it out just yet